src/index.ejs

```diff
@@ -1,25 +1,36 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <script>addEventListener('error', window.__e=function f(e){f.q=f.q||[];f.q.push(e)});</script>
-    <script async src="https://www.google-analytics.com/analytics.js"></script>
-    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
-    <link rel="manifest" href="<%= `${webpackConfig.output.publicPath}manifest.json` %>">
-    <link rel="apple-touch-icon" href="https://process.filestackapi.com/resize=width:144/FmckoTnSQh2QMTsCkPvs">
-    <meta name="apple-mobile-web-app-title" content="Space Force">
-    <meta name="apple-mobile-web-app-capable" content="yes">
-    <meta name="apple-mobile-web-app-status-bar-style" content="black">
-    <meta name="viewport" content="user-scalable=no, width=device-width, initial-scale=1.0, maximum-scale=1.0"/>
-    <style>
-      html, body, #container {
-        width: 100%;
-        height: 100%;
-        margin: 0;
-        overflow: hidden;
-      }
-    </style>
-  </head>
-  <body>
-    <div id="container"></div>
-  </body>
+
+<head>
+  <script>addEventListener('error', window.__e = function f(e) { f.q = f.q || []; f.q.push(e) });</script>
+  <script async src="https://www.google-analytics.com/analytics.js"></script>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Press+Start+2P" rel="stylesheet">
+  <link rel="manifest" href="<%= `${webpackConfig.output.publicPath}manifest.json` %>">
+  <link rel="apple-touch-icon" href="https://process.filestackapi.com/resize=width:144/FmckoTnSQh2QMTsCkPvs">
+  <meta name="apple-mobile-web-app-title" content="Space Force">
+  <meta name="apple-mobile-web-app-capable" content="yes">
+  <meta name="apple-mobile-web-app-status-bar-style" content="black">
+  <meta name="viewport" content="user-scalable=no, width=device-width, initial-scale=1.0, maximum-scale=1.0" />
+  <style>
+    html,
+    body,
+    #container {
+      width: 100%;
+      height: 100%;
+      margin: 0;
+      overflow: hidden;
+    }
+
+    .sceneText {
+      font-family: "Press Start 2P";
+    }
+
+  </style>
+</head>
+
+<body>
+  <div id="container"></div>
+</body>
+
```

Comment:
Same as above. Please keep changes as small as possible.

Can the `font-family` be moved into the component or put on `#container`?